test/functional/helpers/constants/testServer.js

```diff
@@ -0,0 +1,11 @@
+const env = process.env.EDGE_ENV || "int";
+
+const pageName = {
+  int: "alloyTestPage.html",
+  prod: "latestAlloyTestPage.html"
+};
+
+export default {
+  domain: "https://alloyio.com/functional-test",
```

Comment:
Maybe `basePath` instead of `domain` and add a `/` a the end? Any reason to not just concatenate `domain` and `page` together and export the resulting string? It looks like that's what we're doing everywhere we use the information, but maybe you're foreseeing something.

Reply:
Great point! Will change.